addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1123,7 +1123,10 @@ const Wysiwyg = Widget.extend({
                         const toolbarPos = this.toolbar.$el.offset();
                         const colorpickerBottom = toolbarPos.top + this.toolbar.$el.outerHeight() + colorpickerHeight;
                         const clientHeight = this.odooEditor.document.documentElement.clientHeight;
-                        $dropdown[0].classList.toggle('dropup', colorpickerBottom > clientHeight);
+                        const customizePanelOffset = $dropdown.closest('.o_we_customize_panel').offset();
+                        const optionsPanelTop = customizePanelOffset ? customizePanelOffset.top : 0;
```

Comment:
How can `customizePanelOffset`  be falsy?

Reply:
It is the case during the `web_editor > field html > basic > colorpicker` unit test.